src/server/pfs/s3/s3_test.go

```diff
@@ -193,13 +198,17 @@ func TestListBucketsBranchless(t *testing.T) {
 	// should be 0 since no branches have been made yet
```

Comment:
This comment appears out-of-date - also, why did this behavior change?

Reply:
Because these have been switched over to integration tests, all tests run in the same cluster, so the tests aren't isolated. This switch was made because setting up enterprise is easier with integration tests.